core-bundle/src/Migration/Version500/OrderFieldMigration.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Migration\Version500;
+
+use Contao\CoreBundle\Migration\AbstractMigration;
+use Contao\CoreBundle\Migration\MigrationResult;
+use Contao\StringUtil;
+use Doctrine\DBAL\Connection;
+
+/**
+ * @internal
+ */
+class OrderFieldMigration extends AbstractMigration
```

Comment:
Should this be another abstract migration that people could reuse for their tables/fields?

Reply:
Don’t think so, the first `Version410\OrderFieldMigration` is also not abstract.

Reply:
Imho both would make sense, no? :)

Reply:
We should probably create a different PR that adds the abstract migration and adjusts the two existing ones.